Rachis/Rachis/Behaviors/AbstractRaftStateBehavior.cs

```diff
@@ -557,10 +590,27 @@ public virtual AppendEntriesResponse Handle(AppendEntriesRequest req)
                         throw new InvalidOperationException(@"Log entry that is marked with IsTopologyChange should be of type TopologyChangeCommand.
                                                             Instead, it is of type: " + command.GetType() + ". It is probably a bug!");
 
-                    _log.Info("Topology change started (TopologyChangeCommand committed to the log): {0}",
-                        topologyChangeCommand.Requested);
-                    Engine.PersistentState.SetCurrentTopology(topologyChangeCommand.Requested, topologyChange.Index);
-                    Engine.StartTopologyChange(topologyChangeCommand);
+                    
+                    if (topologyChangeCommand.Requested.AllNodes.Contains(Engine.Options.SelfConnection) == false)
+                    {
```

Comment:
Isn't this the same code exactly as above, what is the difference and why do we need both?